sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalV2SessionCatalogSuite.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hive
+
+import org.scalactic.source.Position
+import org.scalatest.Tag
+
+import org.apache.spark.sql.execution.datasources.v2.{V2SessionCatalog, V2SessionCatalogTableBaseSuite}
+import org.apache.spark.sql.hive.test.TestHiveSingleton
+import org.apache.spark.sql.util.CaseInsensitiveStringMap
+
+
+  class HiveExternalV2SessionCatalogTableSuite extends V2SessionCatalogTableBaseSuite
+    with TestHiveSingleton {
+
+  override def newCatalog(): V2SessionCatalog = {
+    val newCatalog = new V2SessionCatalog(spark.sessionState.catalog)
+    newCatalog.initialize("test", CaseInsensitiveStringMap.empty())
+    newCatalog
+  }
```

Comment:
isn't this the same definition as in the base class? am I missing anything?

Reply:
Yes, thanks! I tested different options and didn't see that it's not necesary in the final solution